docs/development.rst

```diff
@@ -0,0 +1,160 @@
+=====================
+Development Standards
+=====================
+
+Front End Development
+=====================
+
+Background
+----------
+
+.. info::
+
+    Consider this the cononical resource for contributing Javascript and CSS. We
+    are currently in the process of modernizing our front end development
+    procedures. You will see a lot of different styles around the code base for
+    front end JavaScript and CSS.
+
+Our modern front end development stack includes the following tools:
+
+* `Gulp`_
+* `Bower`_
+* `Browserify`_
+* `Debowerify`_
```

Comment:
I'd be fine exploring using npm, if it would make a big difference here. I know that npm is not *not* suited for this use case, but it does have short comings.

We aren't using it here yet, but Bower does win out by adding non-JS dependency resolution as well, once we bring those into the stack here w/ use of Less. Again, this doesn't rule out npm, but it seems the larger effort of maintaining these packages by the community has been with bower.